### PR TITLE
erlang 19.3

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by ilovezfs — COMMENTED** [review]
note: we need to wait for the new resources to be posted and the release to be announced before merging this

**Review by ilovezfs — COMMENTED** [review]
release has been announced now

File: Formula/erlang.rb
Comment (fxcoudert) [inline Formula/erlang.rb]: Should the `_clock_gettime` hack be removed now? It seems it was merged.

Comment (ilovezfs) [inline Formula/erlang.rb]: let's find out

**Review by fxcoudert — APPROVED** [review]

**Comment by JCount** [comment]
@ilovezfs a 🍌 for you

**Pull request closed without merging**
